src/event-handler/src/use-cases/CreateEventUseCase.ts

```diff
@@ -23,8 +36,7 @@ export default class {
       return createAuditLogResult
     }
 
-    const createEventResult = await this.api.createEvent(messageId, event)
-
-    return createEventResult
+    const result = await this.api.createEvent(messageId, event)
```

Comment:
A little tidier just to return the unawaited promise here, since we're awaiting it when we call it (I realise it was already like that :-)